repo2docker/__main__.py

```diff
@@ -281,6 +288,8 @@ def make_r2d(argv=None):
         args.run = False
         args.push = False
 
+    r2d.gpus = args.gpus
```

Comment:
```suggestion
    r2d.expose_gpus = args.gpus
```
what do you think of this name? The generic `gpus` is a bit generic. As in, when you see it later in the code you could think that it contains a list of GPU devices or references to GPUs or that it is a flag.